src/item_menu.c

```diff
@@ -455,60 +455,60 @@ extern const u16 gUnknown_0860F074[];
 
 void ResetBagScrollPositions(void)
 {
-    gUnknown_0203CE58.pocket = 0;
+    gUnknown_0203CE58.pocket = ITEMS_POCKET;
     memset(gUnknown_0203CE58.cursorPosition, 0, 10);
     memset(gUnknown_0203CE58.scrollPosition, 0, 10);
 }
 
 void CB2_BagMenuFromStartMenu(void)
 {
-    GoToBagMenu(0, 5, CB2_ReturnToFieldWithOpenMenu);
+    GoToBagMenu(0, POCKETS_COUNT, CB2_ReturnToFieldWithOpenMenu);
 }
 
 void sub_81AABB0(void)
 {
     if (!InBattlePyramid())
-        GoToBagMenu(1, 5, SetCB2ToReshowScreenAfterMenu2);
+        GoToBagMenu(1, POCKETS_COUNT, SetCB2ToReshowScreenAfterMenu2);
     else
         sub_81C4F98(1, SetCB2ToReshowScreenAfterMenu2);
 }
 
 void CB2_ChooseBerry(void)
 {
-    GoToBagMenu(4, 3, CB2_ReturnToFieldContinueScript);
+    GoToBagMenu(4, BERRIES_POCKET, CB2_ReturnToFieldContinueScript);
 }
 
 void sub_81AABF0(void(*callback)(void))
 {
-    GoToBagMenu(5, 3, callback);
+    GoToBagMenu(5, BERRIES_POCKET, callback);
```

Comment:
The first parameter for `GotToBagMenu()` is essentially an index into `gReturnToXStringsTable`, if you're feeling up to documenting that, too.

Reply:
Nice find!

Reply:
Documented.